.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Create a report to help us improve rippled
+title: "[Title with short description] (Version: [rippled version])"
+labels: ''
+assignees: ''
+
+---
+<!-- Please search existing issues to avoid creating duplicates.-->
+<!-- If you are using a formal release, please use the version returned by './rippled --version' as the verison number-->
+<!-- If you are working off of develop, please add the git hash via 'git rev-parse HEAD'-->
+
+## Issue Description
+<!--Describe your bug/feature request in detail.-->
+
+## Steps to Reproduce
+<!--List in detail the exact steps to reproduce the unexpected behavior of the software.-->
+
+## Expected Result
+<!--Explain in detail what behavior you expected to happen.-->
+
+## Actual Result
+<!--Explain in detail what behavior actually happened.-->
+
+## rippled version
+<!--Please provide the version number or the hash to the commit.-->
+
+## Environment
+<!--Please describe your environment setup (such as Ubuntu 18.04 with Boost 1.70).-->
+
+## Supporting Files
+<!--If you have supporting files such as a log, feel free to post a link here using Github Gist.-->
+<!--Consider adding configuration files with private information removed via Github Gist. -->
+<!--Attempt to use this script to remove your private information in the config files https://github.com/ripple/rippled/blob/develop/bin/getInfoRippled.sh -->
```

Comment:
Two TBD's on the `getInfoRippled.sh` we should make sure get done before we merge this template:
1) Add it to the packages
2) Nik tagged the `ips` and `ips_fixed` stanzas for removal. We need to either remove those, or white-list stanzas for the script.
3) We could probably make that script more user friendly as well (the output after a run is a little cryptic).

I'd like to get those done before this is merged. We could also remove the suggestion to use `getInfoRippled.sh` until those changes are made and re-add it when they are.

Reply:
adding this script to packages is simple. At a minimum, we should have manual QA to verify the script works as intended for each release...but if there is also some basic automated test we can add to CI, that would be good too (I'm thinking of a case where we change something about the config or server_info and then the script breaks...)

Reply:
I vote for removing the suggestion until the script is fixed, and for the fix to be switching to a whitelist.

Reply:
The info script could (should?) also log hardware used on the machine, especially CPU type + count as well as (solid state) disk models + RAM available.

Reply:
I will be creating a github issue to track the proposed work and assign it to @mellery451 I will remove this script here for now.

Reply:
https://github.com/ripple/rippled/issues/3248 Created.